src/systems/binds.cpp

```diff
@@ -0,0 +1,158 @@
+#include "binds.h"
+#include "mkb/mkb.h"
+#include "mkb/mkb2_ghidra.h"
```

Comment:
Delete

Reply:
Went through and removed all relative paths and references to mkb2_ghidra (other than in mkb.h of course). I'll try to check before I push, I think an extension I have sometimes adds them back.

Reply:
Same, pretty sure it's the clangd extension. Maybe moving `mkb2_ghidra.h` out of the include paths would fix it